### ISSUE TITLE
Summary metrics don't exist

### ISSUE THREAD

**Issue by jayme-github** [helm/chartmuseum#352]
The summary metrics that are listed in the README.md do not exist:
* chartmuseum_request_duration_seconds
* chartmuseum_request_duration_seconds
* chartmuseum_response_size_bytes

The corresponding total metrics (`*_sum`, `*_count`) do exist, though.

**Comment by scbizu** [comment]
I think it should be with EnableMetrics is on (with the --disablemetrics not set) , chartmuseum uses the prometheus's stdMetrics list , which describes the request_duration_seconds as the [Summary](https://prometheus.io/docs/concepts/metric_types/#summary) type . And you can refer to your $MUSEUM_HOST/metrics to check if it is exist. 

![image](https://user-images.githubusercontent.com/8732481/86960914-de77ca80-c192-11ea-91f0-2b69bd77b232.png)

It is LGTM though.

**Comment by jayme-github** [comment]
Yeah, `sum` and `count` are correctly exposed, but the actual quantiles are missing.

**Comment by scbizu** [comment]
hmm , I see .

It seems that [go-gin-prometheus](https://github.com/zsais/go-gin-prometheus/blob/master/middleware.go#L311) do not support quantile lables by default.

**Comment by jdolitsky** [comment]
cc @davidovich - I think you helped w these metrics?
